AboutMe.Web/Controllers/AccountController.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Application.Dtos.Identity;
+using Core.Constants;
+using Core.Entities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AboutMe.Web.Controllers
+{
+    [AllowAnonymous]
+    public class AccountController : Controller
+    {
+        private readonly SignInManager<User> _signInManager;
+        private readonly UserManager<User> _userManager;
+
+        public AccountController(
+            UserManager<User> userManager,
+            SignInManager<User> signInManager)
+        {
+            _userManager = userManager;
+            _signInManager = signInManager;
+        }
+
+        [HttpGet]
+        public IActionResult Register()
+        {
+            return View();
```

Comment:
I assume this is temporary code, please mark it with TODO comment. If you want to return the Login view here, use 
```
return View("Login")
```